src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -516,5 +521,22 @@ sap.ui.define([
 		return mResponseHeaders;
 	};
 
+	/**
+	 * checks if the _format is part of supported types
+	 * according to fhir all these kinds shall be intrepreted as json
+	 * @param {string} sFormat the format in a particular request
```

Comment:
insert an empty line after the method description

Reply:
done